File: applications/retail-market-app/src/main/java/com/vmware/data/services/demo/cloudNativeData/RetailStoreApp.java
package com.vmware.data.services.demo.cloudNativeData;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

//@ComponentScan
//		(basePackages = {"io.pivotal.pde.demo.cloudNativeData"})
@SpringBootApplication
public class RetailStoreApp
{
	public static void main(String[] args)
	{
		SpringApplication.run(RetailStoreApp.class, args);
	}
}

File: applications/retail-market-app/src/main/kotlin/com/vmware/data/services/demo/cloudNativeData/RetailStoreApp.kt
package com.vmware.data.services.demo.cloudNativeData

import com.vmware.data.services.demo.cloudNativeData.RetailStoreApp
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class RetailStoreApp


    fun main(args: Array<String>) {
        runApplication<RetailStoreApp>(*args)
    }
